diff -ruN orig/CMakeLists.txt new/CMakeLists.txt
--- orig/CMakeLists.txt	2019-06-15 17:17:13.000000000 -0600
+++ new/CMakeLists.txt	2019-06-15 17:18:02.409138484 -0600
@@ -31,6 +31,7 @@
 include(ObsHelpers)
 include(ObsCpack)
 include(GNUInstallDirs)
+add_definitions(-DFFMPEG_MUX_FIXED=\"/usr/libexec/obs-plugins/obs-ffmpeg/ffmpeg-mux\")
 
 # Must be a string in the format of "x.x.x-rcx"
 if(DEFINED RELEASE_CANDIDATE)
diff -ruN orig/plugins/obs-ffmpeg/obs-ffmpeg-mux.c new/plugins/obs-ffmpeg/obs-ffmpeg-mux.c
--- orig/plugins/obs-ffmpeg/obs-ffmpeg-mux.c	2019-06-15 17:17:14.000000000 -0600
+++ new/plugins/obs-ffmpeg/obs-ffmpeg-mux.c	2019-06-15 17:23:06.318162758 -0600
@@ -237,7 +237,11 @@
 		num_tracks++;
 	}
 
+#ifdef FFMPEG_MUX_FIXED
+	dstr_init_copy(cmd, FFMPEG_MUX_FIXED);
+#else
 	dstr_init_move_array(cmd, os_get_executable_path_ptr(FFMPEG_MUX));
+#endif
 	dstr_insert_ch(cmd, 0, '\"');
 	dstr_cat(cmd, "\" \"");
