diff -ruN orig/plugins/CMakeLists.txt new/plugins/CMakeLists.txt
--- orig/plugins/CMakeLists.txt	2021-10-04 11:38:39.000000000 -0600
+++ new/plugins/CMakeLists.txt	2021-10-09 22:39:35.034214131 -0600
@@ -91,3 +91,6 @@
 add_subdirectory(obs-text)
 add_subdirectory(rtmp-services)
 add_subdirectory(text-freetype2)
+add_subdirectory(obs-move-transition)
+add_subdirectory(time-warp-scan)
+add_subdirectory(StreamFX)
diff -ruN orig/UI/frontend-plugins/CMakeLists.txt new/UI/frontend-plugins/CMakeLists.txt
--- orig/UI/frontend-plugins/CMakeLists.txt	2021-10-04 11:38:39.000000000 -0600
+++ new/UI/frontend-plugins/CMakeLists.txt	2021-10-09 22:40:44.586968294 -0600
@@ -1,3 +1,4 @@
 add_subdirectory(decklink-output-ui)
 add_subdirectory(frontend-tools)
 add_subdirectory(decklink-captions)
+add_subdirectory(downstream-keyer)
