diff -ruN orig/plugins/CMakeLists.txt new/plugins/CMakeLists.txt
--- orig/plugins/CMakeLists.txt	2021-12-31 00:32:24.000000000 -0600
+++ new/plugins/CMakeLists.txt	2022-01-02 22:32:56.755256998 -0600
@@ -92,3 +92,6 @@
 add_subdirectory(rtmp-services)
 add_subdirectory(text-freetype2)
 add_subdirectory(aja)
+add_subdirectory(obs-move-transition)
+add_subdirectory(time-warp-scan)
+add_subdirectory(scale-to-sound)
diff -ruN orig/UI/frontend-plugins/CMakeLists.txt new/UI/frontend-plugins/CMakeLists.txt
--- orig/UI/frontend-plugins/CMakeLists.txt	2021-12-31 00:32:24.000000000 -0600
+++ new/UI/frontend-plugins/CMakeLists.txt	2022-01-01 23:14:08.729764327 -0600
@@ -2,3 +2,4 @@
 add_subdirectory(frontend-tools)
 add_subdirectory(decklink-captions)
 add_subdirectory(aja-output-ui)
+add_subdirectory(downstream-keyer)
